### ISSUE TITLE
Add biases to the creation module

### ISSUE THREAD

**Issue by jfcrenshaw** [LSSTDESC/rail_attic#36]
We want to add biases to the creation module, so we can generate data sets with different systematic biases and compare the results of photo-z estimation to the unbiased data set.

We will start with a simple model:
- train a normalizing flow on the unbiased set
- apply a selection function to the unbiased set (e.g. drop galaxies with probability 1 - 1/redshift) to create a biased set
- ~~train a normalizing flow on the biased set~~
- ~~use the biased flow to generate the training set, and the unbiased flow to generate the test set~~

I will produce 3 data products, all as pandas data frames:
- training set: true redshifts and magnitudes of the biased set
- test set, public: true magnitudes of the unbiased set
- test set, private: true redshifts and posteriors of the unbiased set

In the future, we might make this model more sophisticated by using a neural ODE to have continuously parameterized biases.

**Comment by jfcrenshaw** [comment]
Originally, I applied a selection function to created a biased set, then trained a normalizing flow on this set. However, this is redundant. We can generate samples from the unbiased flow, then apply a selection function in the creator to generate the biased sample.

**Comment by jfcrenshaw** [comment]
I built some simple degraders into the creation baseline as part of a `degradation` submodule. I will close this issue. When we want to add more sophisticated degraders, we should make more targeted issues and pull requests
